torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -459,7 +459,8 @@ at::Tensor XLANativeFunctions::_copy_from(const at::Tensor& self,
   auto self_tensor = bridge::TryGetXlaTensor(self);
   if (!self_tensor) {
     static bool sync_update =
-        xla::sys_util::GetEnvBool("XLA_TENSOR_UPDATE_SYNC", true);
+        xla::sys_util::GetEnvBool("XLA_TENSOR_UPDATE_SYNC", true) &&
```

Comment:
It would be helpful, actually important note here why XLA_USE_SPMD condition is added, what it does.

Reply:
Done